src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,6 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
```

Comment:
in `* `you can write only `box-sizing`, nothing else. So move other styles to the appropriate selectors